libs/langchain/langchain/llms/openai.py

```diff
@@ -94,7 +94,7 @@ def _create_retry_decorator(
     import openai
 
     errors = [
-        openai.error.Timeout,
+        openai.Timeout,
```

Comment:
Thank you for reviewing this PR. Just to confirm. are you suggesting that we should not update to openai v1.0.0? This would mean that the error we are receiving that is documented in #11755 that is solved by updating to openai v1.0.0 (https://github.com/openai/openai-python/issues/574) will still persist.

Reply:
Hello @baskaryan , any updates? Just wanted to confirm that, as you mentioned, langchain is not looking to update to openai v1.0.0.